record_tx_strategy_outgoing_tx.go

```diff
@@ -10,9 +10,11 @@ import (
 )
 
 type outgoingTx struct {
-	Hex            string
+	BtTx           *bt.Tx
 	RelatedDraftID string
 	XPubKey        string
+
+	txId string
```

Comment:
Minor - lint error:
`var-naming: struct field txId should be txID (revive)go-golangci-lint`

Reply:
Done